LTHPasscodeViewController/LTHPasscodeViewController.m

```diff
@@ -57,6 +57,15 @@
 #define LTHMainWindow [UIApplication sharedApplication].windows.firstObject
 #endif
 
+#import "MEGAChatSdk.h"
+#ifdef MNZ_SHARE_EXTENSION
+#import "MEGAShare-Swift.h"
+#elif MNZ_PICKER_EXTENSION
+#import "MEGAPicker-Swift.h"
+#else
+#import "MEGA-Swift.h"
+#endif
```

Comment:
Why we need to import our class into a third party library? it is we to depend on a 3rd party library, not the other way around.

Reply:
I'm using UIColor.mnz_green00A886. And it need #import "MEGA-Swift.h"
Should I make the color the other way and remove the import?

Reply:
You can create the color inside this library. Even hardcode it is OK.

Reply:
Just imagine this is an open source library, which should not aware any mega things.

Reply:
Got it. Just update